### PR TITLE
WEBAPP-210: Add feedback to search results

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/routes/feedback/components/Feedback.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ const SubmitButton = styled(CleanLink)`
   border-radius: 0.25em;
 `
 
-export type FeedbackDropdownType = {
+type FeedbackDropdownType = {
   value: string,
   feedbackType: string | null,
```
Comment (maxammann) [inline src/routes/feedback/components/Feedback.js]: You can use `feedbackType: ?string`

File: src/routes/search/containers/SearchPage.js [hunk 2]
```diff
@@ -54,6 +83,50 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
 
   onFilterTextChange = (filterText: string) => this.setState({filterText: filterText})
 
+  renderFeedback = (categories: Array<CategoryListItemType>) => {
```
Comment (maxammann) [inline src/routes/search/containers/SearchPage.js]: Probalby can be extracted to a separate component.

File: src/routes/search/containers/SearchPage.js [hunk 3]
```diff
@@ -54,6 +83,50 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
...
+          )}
+          <FeedbackModal
+            query={filterText}
+            city={city}
+            cities={cities}
+            language={language}
+            route={route}
+            pathname={pathname}
+            isPositiveRatingSelected={feedbackType === POSITIVE_RATING}
+            isOpen={feedbackType !== undefined}
```
Comment (maxammann) [inline src/routes/search/containers/SearchPage.js]: Why do you need to check for undefined here?

I think we shouldn't make a difference between null and undefined in our code. So just go with !!feedbackType

File: src/routes/search/containers/SearchPage.js (hunk 2 above)
Comment (klinzo) [reply src/routes/search/containers/SearchPage.js]: Done

**Comment by maxammann** [comment]
Also I think the used font for the questions should be Open Sans

**Comment by maxammann** [comment]
Also the dropdown behaves a bit weird:

If you click on one element and do not move the cursor, then the cursor still looks like "the pointing finger".

**Comment by maxammann** [comment]
Design: I  think the fading animation is a bit too much. It should be quicker or not used.

**Comment by klinzo** [comment]
This just happens in the dev console, doesn't it? I've never seen the feedback box looking like that outside.

**Comment by maxammann** [comment]
@klinzo I think it happens if the browser window looses focus. Do we know why this happens?

**Comment by klinzo** [comment]
@maxammann Right. No, I don't know why it happens. I'll create a task for this and the cursor thing. I'll also create a task for the font, since openSans looks a bit weird imo. Regarding the fading animation, there isn't one for me and I also didn't implement one.

**Comment by maxammann** [comment]
2018-07-31-10 48 41-screenshot

There is one.

**Comment by klinzo** [comment]
@maxammann This is the transition of the toolbar which was there before.

**Pull request merged**